packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -128,6 +169,43 @@ export class AzureClient {
         return { container: fluidContainer, services };
     }
 
+    /**
+     * Get the list of versions for specific container
+     * @param id - Unique ID of the source container in Azure Fluid Relay.
+     * @param maxCount - Max number of versions to retreive,
+     * @returns Array of available versions
+     */
+     public async getContainerVersions(
+        id: string,
+        maxCount: number,
```

Comment:
I prefer direct props over a props objects in scenarios where we think we will not be modifying the API. Have you considered other optional properties we may extend to in the future?

Reply:
For some sort of pagination purposes, we could consider `startVersion`. That seems like a likely option. Or some sort of time/date based filtering. We could fold this parameter under options, and extend from there. I'll make the change.

Reply:
I have folded maxCount under AzureGetVersionsOptions. More on that in the other discussion.